Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
+O(1)
 
 2. What is the space complexity of your ring buffer's `append` function?
+o(1)
 
 3. What is the runtime complexity of your ring buffer's `get` method?
+O(n)
 
 4. What is the space complexity of your ring buffer's `get` method?
-
+O(n)
 
 5. What is the runtime complexity of the provided code in `names.py`?
+O(n^2)
 
 6. What is the space complexity of the provided code in `names.py`?
+O(n^2)
```

Comment:
Since this code takes in 2 arrays and loops through them without creating copies or new elements in memory, we can consider this O(1) space complexity.